ansible-playbook-base/roles/common/tasks/setup_10g_nic.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- hosts: servers
+  tasks:
+    - name: Configures the first 10G nic with an active link
```

Comment:
This comment talks about plumbing this into the playbook:

- So, currently this is run as an independent play, which would need to be integrated to the global playbook and roles (site.yml and possibly the common role)
- This configuration is setup specific, which means a setup-profile declares intent that it needs an IPv4 address configured with said parameters
- The above part of the plumbing is achieved, by adding a variable to each host in the setup-profile (for such a setup), lets use the same naming convention as the IPoIB and call this, "confipv4addr" (so this will be of the form confipv4addr: <IPAddr>
- The presence of this variable should trigger the configuration play setup_10g_nic.yaml, which can be achieved using a similar pattern to the check, https://github.com/gluster/gbench/blob/master/ansible-playbook-base/roles/common/tasks/main.yml#L16

The above would complete the required plumbing to run this play as part of the playbook, when a setup deals with only IPv4 addresses, and needs one setup for it.